scalameta/trees/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -518,8 +518,9 @@ object TreeSyntax {
 
       // Multiples and optionals
       implicit def syntaxArgs: Syntax[Seq[Term.Arg]] = Syntax {
-        case (b: Term.Block) :: Nil => s(" ", b)
-        case args                   => s("(", r(args, ", "), ")")
+        case (b: Term.Block) :: Nil    => s(" ", b)
+        case (f: Term.Function) :: Nil => s(" { ", f, " }")
```

Comment:
Should we maybe restrict this to Term.Function where all params have a defined `decltpe`?

```scala
// OK
@ List(1).map(implicit a => a + 2)
res24: List[Int] = List(3)
// Not OK
@ List(1).map(implicit a: Int => a + 2)
<console>:1: identifier expected but integer literal found.
List(1).map(implicit a: Int => a + 2)                                   ^
```